codegen/core/src/main/java/software/amazon/smithy/python/codegen/writer/ImportDeclarations.java

```diff
@@ -1,34 +1,26 @@
 /*
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at
- *
- *  http://aws.amazon.com/apache2.0
- *
- * or in the "license" file accompanying this file. This file is distributed
- * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
+ * SPDX-License-Identifier: Apache-2.0
  */
-
-package software.amazon.smithy.python.codegen;
+package software.amazon.smithy.python.codegen.writer;
 
 import java.util.Iterator;
 import java.util.Map;
 import java.util.TreeMap;
 import software.amazon.smithy.codegen.core.CodegenException;
 import software.amazon.smithy.codegen.core.ImportContainer;
 import software.amazon.smithy.codegen.core.Symbol;
+import software.amazon.smithy.python.codegen.CodegenUtils;
+import software.amazon.smithy.python.codegen.PythonSettings;
+import software.amazon.smithy.python.codegen.SymbolProperties;
 import software.amazon.smithy.utils.SmithyInternalApi;
 import software.amazon.smithy.utils.StringUtils;
 
 /**
  * Internal class used for aggregating imports of a file.
  */
 @SmithyInternalApi
-final class ImportDeclarations implements ImportContainer {
+public final class ImportDeclarations implements ImportContainer {
```

Comment:
Why does this need to be public?

Reply:
Used by `GenerationContext`

Reply:
It doesn't, changing.

Reply:
Used in `ConfigGenerator` a la `writer.getImportContainer().addImport`